src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React from 'react';
+import NavBar from './components/navbar';
+import Main from './components/main' ;
+import GridItems from './components/griditems';
+import './style.css';
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="wrapper">
+    <NavBar/>
+    <Main/>
+    <GridItems/>
```

Comment:
Create a pages directory and create a HomePage directory
HomePage.jsx
and call these components inside homePage
Inside this file import the HomePage
```suggestion
    <div className="wrapper">
    <HomePage />
    <GridItems/>
```